packages/ios/Metriport-Health/Package.swift

```diff
@@ -0,0 +1,29 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Metriport-IOS",
+    platforms: [.iOS(.v13)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Metriport-IOS",
+            targets: ["Metriport-IOS"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
+            name: "Metriport-IOS",
+            dependencies: []),
+        .testTarget(
+            name: "Metriport-IOSTests",
+            dependencies: ["Metriport-IOS"]),
```

Comment:
This is now called Metriport-Health, but everything else references Metriport-IOS, including some sub-dirs. Is there a proper rename that we should do?

Also, I'd vouch for naming this `metriport-sdk` but not a big deal